packages/api/src/schema/schema.yaml

```diff
@@ -211,10 +211,14 @@ components:
               - multistream.disconnected
               - playback.user.new
               - asset.created
-              - asset.status
-              - task.created
-              - task.status
-              - task.finished
+              - asset.updated
+              - asset.failed
+              - asset.ready
+              - asset.deleted
```

Comment:
Need to fire this from `app.deleted("/:id")` in `asset.ts`

Reply:
Still missing this.

Reply:
Added!